website/pages/partials/envvars.mdx

```diff
@@ -31,7 +31,7 @@
         <code>NOMAD_SECRETS_DIR</code>
       </td>
       <td>
-        Path to the task's secrets directory. See
+        Path to the task&lsquo;s secrets directory. See
```

Comment:
Are these `&lsquo;` part of a linter rule? Are they automatically applied?

Reply:
Yes they are, and they are not automatically applied. I am a little annoyed by them and a few others on the team are as well, but it was a recent addition to the eslint react defaults, so I'm trusting that it's there for a good reason 😕